### DISCUSSION TITLE
Back Link from Footnotes Broken After Conversion

### DISCUSSION THREAD

**Discussion by epact-flair** [apprenticeharper/DeDRM_tools#2194] [Q&A]
Hi,

I converted an E-Book from a very popular Epub format to DOCX & ODT.

EVERYTHING came over except...the work has extensive footnotes.  

The jump-TO the footnotes works GREAT!

But,..clicking on the corresponding link to jump you BACK to the text - doesn't work.

I can SEE the text.  

And, there's a link.  The text has an underline.  Different color.  

Mouse changes shape as if to indicate, "Yep!  Link here.  Ready to go!", **but**...clicking does nothing.

What do you think is happening?

Sincerely,

EFP

**Comment by ElleKayEm** [comment]
For non-DRM issues, I suggest asking over at MobileRead's calibre forum.

**Comment by epact-flair** [comment]
Hello ElleKayEm,

Got it! Thank you for your reply!!

EFP
